brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-2658 Fix: Modal - On click on overlay the background to scroll appear when `Close on Overlay Click` is disabled.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-2658

### Screenshots

- 

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Modal and other multiple content on page.
- Publish the page, Visit frontend, click on modal.
- Modal is open, now click on overlay scroll should not be visible.
- Clicking on Close button only Modal will close.
- Now try Close on overlay click/ esc key press option.
- In case of Close on overlay click - Modal will be close on click on overlay.
- And in other case like esc key press - Modal will be close on esc key press and clicking on scroll Icon. 

### Checklist:
